addon/lib/utils/register-helper.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+function registerHelperIteration1(name, helperFunction) {
+  //earlier versions of ember with htmlbars used this
+  Ember.HTMLBars.helpers[name] = helperFunction;
```

Comment:
I think I can do that, at least try. Though I have no idea how do you want this micro-lib (an ember-cli addon itself ?)